contents/blog/clickhouse-announcement.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2021-08-13
+title: We just made PostHog Open Source 1000x more scalable via ClickHouse
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+featuredImage: ../images/blog/filename.png
+featuredImageType: full
+---
+We're excited to announce a major under-the-hood upgrade for PostHog Open Source! Over the past few weeks, our team have been hard at work moving over from Postgres to ClickHouse, the same database that powers PostHog Cloud.
+
+This means that anyone can now get started using PostHog for free at scale - you'll no longer be limited to 10k monthly active users. Read on to find about more about the benefits and how we did this!
```

Comment:
I think we'd want to rewrite this sentence just to avoid any confusion. We're talking about PostHog Open Source, so saying 'use PostHog for free at scale' is a bit confusing when we previously offered PostHog Free and currently offer PostHog Scale as separate products. 

"This means you can now use PostHog Open Source for free without being limited to 10k monthly active users."